app/models/user.rb

```diff
@@ -1,7 +1,8 @@
 class User < ActiveRecord::Base
   has_many :wikis
   has_many :collaborations
-  has_many :private_wikis, through: :collaborations
+  has_many :private_wikis, through: :collaborations, source: :wiki
+  
```

Comment:
`Collaboration` only knows about wikis, so this line actually defines an alias, `private_wikis`, for a collaboration's wikis attribute.